cluster_experiments/power_config.py

```diff
@@ -122,6 +126,88 @@ class PowerConfig:
 
     seed: Optional[int] = None
 
+    def __post_init__(self):
+        if "switchback" not in self.splitter:
+            if self._are_different(self.switch_frequency, None):
+                self.switch_frequency = None
+                logging.warning(
+                    f"{self.switch_frequency = } has no effect"
+                    f"with {self.splitter = }. Overriding switch_frequency to None."
+                )
+            if self._are_different(self.washover_time_delta, None):
+                self.washover_time_delta = None
+                logging.warning(
+                    f"{self.washover_time_delta = } has no effect"
+                    f"with {self.splitter = }. Overriding washover_time_delta to None."
+                )
+            if self._are_different(self.washover, ""):
+                self.washover = ""
+                logging.warning(
+                    f"{self.washover = } has no effect with {self.splitter = }."
+                    'Overriding washover to "".'
+                )
+            if self._are_different(self.time_col, None):
+                self.time_col = None
+                logging.warning(
+                    f"{self.time_col = } has no effect with {self.splitter = } "
+                    "Splitter. Overriding time_col to None."
+                )
+
+        if self.perturbator not in {"normal", "beta_relative_positive"}:
+            if self._are_different(self.scale, None):
+                self.scale = None
+                logging.warning(
+                    f"{self.scale = } has no effect with {self.perturbator = }."
+                    "Overriding scale to None."
+                )
+
+        if "balanced" not in self.splitter:
+            if self._are_different(self.treatments, None):
+                self.treatments = None
+                logging.warning(
+                    f"{self.treatments = } has no effect with {self.splitter = }."
+                    "Overriding treatments to None."
+                )
+
+        if "stratified" not in self.splitter and "ttest" not in self.analysis:
```

Comment:
We can even be more precise and do:
```suggestion
        if "stratified" not in self.splitter and "paired_ttest" not in self.analysis:
```

Reply:
thanks, I changed it